### PR TITLE
[LIB-300] Connection monitor

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: src/pinger.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      })
+    },
+
+    startMonitoring() {
+      this.interval = setInterval(() => this.ping(), this.timeout);
+    },
+
+    ping() {
+      this.request()
+        .then(() => this.emit('connect'))
```
Comment (dancio) [inline src/pinger.js]: `connected` / `disconnected` ? and do we wan to emit e.g  `connect` in loop many times (each 5 sec)? or just when something has changed?

**Comment by dancio** [comment]
besides minor things lgtm

File: src/pinger.js (hunk 1 above)
Comment (jakubbilko) [inline src/pinger.js]: 4207487b64c9bcfb6e10ff9c5922b3c7cb47081d

**Comment by dancio** [comment]
lgtm 2

**Pull request merged**